--- C:\xampp\htdocs\stp\sites\all\modules\contrib\quickedit\js\views\EditorView.js 
+++ C:\Users\Debraj.Naik\Desktop\quickedit\js\views\EditorView.js 
@@ -222,16 +222,11 @@
         $form.prop('novalidate', true);
         var $submit = $form.find('.quickedit-form-submit');
         self.formSaveAjax = Drupal.quickedit.util.form.ajaxifySaving(formOptions, $submit);
-        
+
         function removeHiddenForm () {
           Drupal.quickedit.util.form.unajaxifySaving(self.formSaveAjax);
           delete self.formSaveAjax;
           $backstage.remove();
-          
-           
-           location.reload(true);//edited by debraj to reload the pages
-           //self.location.refresh();
-          
         }
 
         // Work-around for https://drupal.org/node/2019481 in Drupal 7.
